Engineering thesis/rozdzialy/Implementacja.tex

```diff
@@ -0,0 +1,726 @@
+Celem rozdziału jest przedstawienie środowiska i bibliotek umożliwiających stworzenie aplikacji webowej z zastosowaniem urządzenia Raspberry Pi.
+
+\section{Środowisko programistyczne}
+W celu realizacji założeń projektowych, program został wykonany przy użyciu środowiska programistycznego Visual Studio Code. JavaScript jest językiem programowym, który jest popularny wśród programistów i istnieje wiele gotowych bibliotek i frameworków, które można wykorzystać do tworzenia aplikacji. Visual Studio Code jest środowiskiem, które ma dobrą obsługę JavaScriptu i Node.js, gwarantując narzędzia do podpowiadania składni i automatycznego uzupełniania kodu IntelliSense, a także narzędzia do debugowania i testowania aplikacji. Visual Studio Code również umożliwia korzystanie z niego na różnych systemach operacyjnych takich jak Linux czy Windows. Raspberry Pi ma wbudowany system operacyjny Linux dzięki czemu środowisko to idealnie się wkomponuje w tworzeniu aplikacji na tym urządzeniu. Aplikacja składa się z dwóch głównych elementów. Jest to serwer i klient. Część serwerowa to część aplikacji, która działa na serwerze i odpowiada za przetwarzanie i przechowywanie danych oraz obsługę żądań użytkowników. Została napisana w języku programowania JavaScript przy użyciu Node.js. Część klienta to część aplikacji, która działa na urządzeniu użytkownika i odpowiada za wyświetlenie interfejsu użytkownika oraz komunikację z serwerem. Została ona napisana również w języku programowania JavaScript przy użyciu biblioteki React.js. Visual Studio Code pozwala na łatwą obsługę i zarządzanie strukturą plików co będzie pomocne przy tak dużych zasobach drzewa programu.
```

Comment:
```
W celu realizacji założeń projektowych, program został wykonany przy użyciu środowiska programistycznego Visual Studio Code
```
VSC nie jest środowiskiem programistycznym. Jest on edytorem. Twój program został stworzony przy użyciu języka JavaScript oraz środowiska uruchomieniowego Node.JS. Przy tworzeniu kodu opierałeś się VSC.

Reply:
JavaScript jest językiem programowym, -> JavaScript jest językiem programowania

Reply:
```
JavaScript jest językiem programowym, który jest popularny wśród programistów i istnieje wiele gotowych bibliotek i frameworków, które można wykorzystać do tworzenia aplikacji.
```
1. zdanie wielokrotnie złożone, trudne do przeczytania. Rozbij to proszę na co najmniej 2 zdania.
2. Uzasadnienie że użyłeś JSa jest takie że jest popularny wśrod programistów - warto to chyba przemyśleć. Co chciałęś przekazać tym zdaniem?
3. czym jest framework? To nie po polsku.

Reply:
```
Visual Studio Code jest środowiskiem, które ma dobrą obsługę JavaScriptu i Node.js, gwarantując narzędzia do podpowiadania składni i automatycznego uzupełniania kodu IntelliSense, a także narzędzia do debugowania i testowania aplikacji.
```
1. Zdanie wielokrotnie złożone. Rozbij to na co najmniej 3 proste zdanie.
2. VSC jest edytorem - nie jest środowiskiem. 
3. "Ma dobrą obsługę JavaScriptu" brzmi jak coś co możesz powiedzieć do kolegi a nie jak do profesora. Proszę o zmianę.
4. To dodatki do VSC zapewniają podpowiedzi do składni oraz narzędzia do debugowania

Reply:
zmieniłem cel rozdziału na : Celem rozdziału jest przedstawienie środowiska programowania i bibliotek umożliwiających implementację kodu potrzebnych do stworzenia aplikacji webowej z zastosowaniem urządzenia Raspberry Pi.

Reply:
Nazwę rozdziału zamiast środowisko programowania zmieniłem na edytor kodu źródłowego i poskracałem zdania